backrest/docker-compose.yaml

```diff
@@ -0,0 +1,30 @@
+services:
+  backrest:
+    image: garethgeorge/backrest:latest
+    volumes:
+      - data:/data
+      - config:/config
+      - cache:/cache
+      - backrest-restores:/backrest-restores
```

Comment:
`      - backrest-restores:/backrest-restores`

Won't this throw an error if use_external_volume is false? In the instance yaml it is blocked by this conditional, so the volume config might not exist in the final override.yaml